src/pages/settings/Wallet/assignedCardPropTypes.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import CONST from '../../../CONST';
+
+/** Assigned Card props */
+const assignedCardPropTypes = PropTypes.shape({
+    cardID: PropTypes.number,
+    state: PropTypes.number,
+    bank: PropTypes.string,
+    availableSpend: PropTypes.number,
+    domainName: PropTypes.string,
+    lastFourPAN: PropTypes.string,
+    cardName: PropTypes.string,
+    isVirtual: PropTypes.bool,
+    fraud: PropTypes.oneOf([CONST.EXPENSIFY_CARD.FRAUD_TYPES.DOMAIN, CONST.EXPENSIFY_CARD.FRAUD_TYPES.USER, CONST.EXPENSIFY_CARD.FRAUD_TYPES.NONE]),
+    cardholderFirstName: PropTypes.string,
+    cardholderLastName: PropTypes.string,
+    errors: PropTypes.objectOf(PropTypes.string),
+});
```

Comment:
cc @hayata-suenaga what's the new pattern for these?

see https://github.com/Expensify/App/blob/main/contributingGuides/PROPTYPES_CONVERSION_TABLE.md#important-considerations

Reply:
Nevermind, this isn't a ts file yet. Ignore me